File: chess.py
# encoding definition
"""coding=utf-8"""
from colorama import Fore, Back, Style


# ChessBoard function definition
def ChessBoard(userColumns, userRows):
    board = ''
    # index for control lines
    for i in range(userRows):
        # index for control characters in lines
        for j in range(userColumns):
            if (i + j) % 2 == 0:
                board += Back.WHITE + ' ' + Back.RESET
            else:
                board += Back.BLACK + ' ' + Back.RESET
        board += '\n'
    print(board)


# input loop
while True:
    # script description
    print(Fore.RED + '\n=========================================' + Fore.YELLOW + 'v1.0.170508' + Fore.RESET)
    print(Fore.CYAN + 'This Python2/Python3 script will draw a chess board\nusing the following inputs...' + Style.DIM + Fore.WHITE + '\n(For exit, enter 0 in any of the inputs.)' + Fore.RESET + Style.RESET_ALL)
    print(Fore.BLUE + '----------------------------------------------------' + Fore.RESET)
    # user input prompt and evaluation
    try:
        userColumns = int(input(Fore.YELLOW + 'Enter a value for the number of COLUMNS: ' + Fore.RESET))
        userRows = int(input(Fore.YELLOW + 'Enter a value for the number of ROWS: ' + Fore.RESET))
    except Exception:
        print('\n' + Back.RED + ' *** INPUT ERROR ***' + Back.RESET + '\n' + Back.RED + ' Input value of Columns and Rows should be a Number. ' + Back.RESET)
        continue
    print(Fore.RED + '====================================================\n' + Fore.RESET)

    # user input evaluation
    if userColumns == 0 or userRows == 0:
        print(Back.MAGENTA + '-----------------' + Back.RESET)
        print(Back.MAGENTA + '|' + Back.RESET + Style.BRIGHT + '  Good Bye!!!  ' + Style.RESET_ALL + Back.MAGENTA + '|' + Back.RESET)
        print(Back.MAGENTA + '-----------------' + Back.RESET + '\n')
        break
    elif userColumns < 3 or userRows < 3:
        print(Back.RED + ' *** INPUT ERROR *** ' + Back.RESET + '\n' + Back.RED + ' Input value of Columns and Rows should not be less than 3 units. ' + Back.RESET)
    else:
        # board outputs
        ChessBoard(userColumns, userRows)
        break

File: chess.js
// npm package require
var prompt = require('prompt-sync')()
var colors = require('colors')

// class definition
function ChessBoard (cols, rows) {
  this.cols = cols
  this.rows = rows
  var board = ''

  // index for control lines
  for (var i = 0; i < this.rows; i++) {
    // index for control characters in lines
    for (var j = 0; j < this.cols; j++) {
      if ((i + j) % 2 === 0) {
        board += ' '.bgWhite
      } else {
        board += ' '.bgBlack
      }
    }
    board += '\n'
  }
  return board
}

// input loop
while (true) {
    // script description and user input prompt
  console.log('\n================================================'.red + 'v1.0.170508'.yellow)
  console.log('This JS script will draw a chess board using the following\ninputs...'.cyan + '(For exit, press 0 or Enter in any of the inputs.)'.gray)
  console.log('-----------------------------------------------------------'.blue)
  var userColumns = Number(prompt('Enter a value for the number of COLUMNS: '.italic.yellow))
  var userRows = Number(prompt('Enter a value for the number of ROWS: '.italic.yellow))
  console.log('===========================================================\n'.red)

  // user inputs evaluation
  if (isNaN(userColumns) || isNaN(userRows)) {
    console.log('*** INPUT ERROR ***'.bgRed + '\n' + 'Input value of Columns and Rows should be a Number.'.bgRed)
  } else if (userColumns === 0 || userRows === 0) {
    console.log('-----------------'.bgMagenta)
    console.log('|'.bgMagenta + '  Good Bye!!!  ' + '|'.bgMagenta)
    console.log('-----------------'.bgMagenta + '\n')
    break
  } else if (userColumns < 3 || userRows < 3) {
    console.log('*** INPUT ERROR ***'.bgRed + '\n' + 'Input value of Columns and Rows should not be less than 3 units.'.bgRed)
  } else {
    // call ChessBoard() class
    var userBoard = ChessBoard(userColumns, userRows)

    // draw board
    console.log(userBoard)
    break
  }
}
